osi_common.proto

```diff
@@ -387,6 +387,125 @@ message LogicalLaneAssignment
     optional double angle_to_lane = 4;
 }
 
+// \brief A bounding box containing a sub-section of a object. 
+//
+// A bounding box representing a sub-section of it's parents overall dimension, 
+// either that of a \c MovingObject or \c StationaryObject .
+//
+// The parent frame of the \c BoundingBox is not relative to the parent object
+// it is associated to, but in the same parent frame as the parent object.
```

Comment:
I don't understand this sentence, could it be reworded?

Reply:
What's actually meant is that a sub bounding box is not relative to its parent bounding box but rather to the parent frame of its parent bounding box. It is in the same frame as its parent object. E.g. if the main bounding box is in global coordinates, the sub bounding box is as well.

But I definitely agree, that it is hard to understand. Could also be extended by an example.